src/pytest_inmanta_lsm/lsm_project.py

```diff
@@ -27,6 +27,12 @@
     "    - If you are using v2 modules: make sure the inmanta-module-lsm is installed in your venv."
 )
 
+# Try to import from inmanta.util.dict_path, if not available, fall back to the deprecated inmanta_lsm.dict_path
+try:
+    from inmanta.util import dict_path
+except ImportError:
+    from inmanta_lsm import dict_path
+
```

Comment:
@sanderr is this what you meant with the try catch? also I don't think it deserves a change entry, does it?

Reply:
Indeed. Also agreed on the change entry part. Did you verify that the latest iso6 has dict_path in inmanta-lsm?

Reply:
You are talking about iso6-stable right? It does indeed have the dict path in inmanta-lsm